easybuild/easyconfigs/h/Hadoop/Hadoop-2.5.0-cdh5.3.1-native.eb

```diff
@@ -26,7 +26,7 @@ dependencies = [(java, javaver)]
 
 build_native_libs = True
 
-extra_native_libs = [('snappy', 'lib/libsnappy.so')]
+extra_native_libs = [('snappy', 'lib/libsnappy.so*')]
```

Comment:
yeah, basically the easyblocks become a distraction in relation to easyconfigs; 
it would be ideal if there was a way to have both of them next to each other
(I even consider the crazy idea of passing easyblocks in docstrings inside easyconfigs... :-P )